rust/core-lib/src/plugins/catalog.rs

```diff
@@ -116,7 +116,6 @@ fn find_all_manifests(paths: &[PathBuf]) -> Vec<PathBuf> {
             dir.flat_map(|item| item.map(|p| p.path()).ok())
                 .map(|dir| dir.join("manifest.toml"))
                 .filter(|f| f.exists())
-                .for_each(|f| manifest_paths.push(f.to_owned()))
```

Comment:
This is a mistake; we do need this line, we just can replace `f.to_owned()` with `f`.

Reply:
:ok_hand: Sorry it's my wrong.